src/components/Modal/AddToFolder.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+import Modal from "./Modal";
+import styles from "./addToFolder.module.scss";
+import checkIcon from "../../image/check.svg";
+
+interface Category {
+  id: number;
+  name: string;
+  link?: { count: number };
+}
+
+interface AddToFolderProps {
+  isOpen: boolean;
+  onClose: () => void;
+  categoryList: Category[];
```

Comment:
@HaeJungg 

```suggestion
  categories: Category[];
```
이런 경우 해당 요소의 복수형을 사용하는게 조금 더 관리하기 좋아보입니다!